app/views/data_view.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+from app.models import Donor, Donation, Item
+from django.http import HttpResponse, JsonResponse, HttpResponseBadRequest
+from django.views import View
+import simplejson as json
+
+# return aggregate value for given time interval
+# if request.GET['interval'] is false, return value of all items
+@login_required(login_url='/login')
+def aggregate_value(request):
+    return_value = 0
+    if request.GET['interval'] is False:
+        return_value = sum(Item.value for item in Item.objects.all())
```

Comment:
This line will not work. `Item.value` => `item.value` might even be `item.value()`. Check using the `python manage.py shell`